packages/build-utils/test/unit.convert-runtime-to-plugin.test.ts

```diff
@@ -68,12 +81,12 @@ describe('convert-runtime-to-plugin', () => {
       server: {
         pages: {
           api: {
-            'index.py': expect.stringContaining('index'),
+            'index.py': expect.stringContaining('handler'),
             'index.py.nft.json': expect.stringContaining('{'),
             users: {
-              'get.py': expect.stringContaining('get'),
+              'get.py': expect.stringContaining('handler'),
               'get.py.nft.json': expect.stringContaining('{'),
-              'post.py': expect.stringContaining('post'),
+              'post.py': expect.stringContaining('handler'),
```

Comment:
Is this change necessary? Looks like a bug to me since its not reading the contents

Reply:
Yes, because the files placed in `server/pages/api` should be the actual Lambda entrypoints, and therefore the launcher files, not the request handlers that the user provided in their source code (those are separate files invoked by the launcher, depending on what Legacy Runtime is used).